### PR TITLE
Basic html and javascript UI implementations

### PR DESCRIPTION
**Pull request by alexiscyber14**
Implement only a basic UI with plain HTML:
Create a collection that keeps a list of books (hint: you can use an array of objects for that).
Create a function to add a new book to the collection, with title and author.
Create a function to remove a book from the collection (hint: you can use the [array filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method).
Display all books saved in the collection in the top part of the page.
Make sure that when a user clicks the "Add" button:
A new book is added to the collection.
The new book is displayed in the page.
Make sure that when a user clicks the "Remove" button:
The correct book is removed from the collection.
The correct book dissapears from the page.
Make sure that data is preserved in the browser's memory by using [localStorage]

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @alexiscyber14,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly collaborate with your coding partner on this project. You also do not have a readme file. Kindly add a readme file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @   ✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  }
+}
+
+
+// Add a new book
+function addBook(title, author) {
+    books.push({ title, author });
+    localStorage.setItem("books", JSON.stringify(books));
+    renderBooks();
+  }
```
[the remark this answers is not in the document]
Comment (alexiscyber14) [reply index.js:27]: Thanks man. i hadn't thought about that but lemme fix it

**Review by prolajumokeoni — APPROVED** [review]
Hi @alexiscyber14 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- All changes requested have been implemented 🥇  


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
- [ ] [Branch name](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects) should have a name of a feature or action related to your project eg. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**